Task/task11_18week.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
checklist
- [ ] GIS Day talk prep
- [ ] literature review
- [ ] research tasks
- [ ] Paper 3 for Theory
- [ ] 30 day map chanllenges

Reply:
add those info there